### ISSUE TITLE
Stable Diffustion

### ISSUE THREAD

**Issue by Tobe2d** [deepinsight/insightface#2298]
Is there any way to use this with Stable Diffusion?

**Comment by gokhanbaydar** [comment]
I wanted to add this as a plugin to AUTO1111 but I cannot find their inswapper_128.onnx model file anywhere, seems to be vanished, if anyone can help me with that file I could start working on a plugin.

**Comment by idinkov** [comment]
I have found the file here: https://drive.google.com/drive/folders/1X7fiFdNJQ0xAg-gOS1YO38qmFLnWRmd2

**Comment by firrrari** [comment]
> I have found the file here: https://drive.google.com/drive/folders/1X7fiFdNJQ0xAg-gOS1YO38qmFLnWRmd2

What do i download? and what do i do with this?

**Comment by idinkov** [comment]
This is for @gokhanbaydar as he said he needed the file.

**Comment by gokhanbaydar** [comment]
> I have found the file here: https://drive.google.com/drive/folders/1X7fiFdNJQ0xAg-gOS1YO38qmFLnWRmd2

Thanks! Started working on a plugin.. I will update here as well when I release it

**Comment by BAIS1C** [comment]
of course, you can just upload it to a discord channel with the insightface app and swap faces there...

**Comment by stavsap** [comment]
this makes the face 128X128, is there higher resolution model? or some settings?

**Comment by sciencefreak500** [comment]
Very curious of your progress @gokhanbaydar regarding the plugin.

**Comment by dogfight-mike** [comment]
I too am very curious how you are doing with a Stable Diffusion compatible extension. This stuff is magic.

**Comment by newbie-lad** [comment]
Thing about Stable diffusion is a simple one:
you either _code it yourself_, or you don't :D
There is _a reason_ proprietary solutions are in demand.
Although, open source is loveable :)

**Comment by gokhanbaydar** [comment]
Dropped this since roop came out, check here if you are interested (not my repo): https://github.com/s0md3v/sd-webui-roop/
